lua/nvim-treesitter/configs.lua

```diff
@@ -20,7 +20,7 @@ local M = {}
 local config = {
   modules = {},
   sync_install = false,
-  ensure_installed = {},
+  ensure_installed = { "lua", "query", "vim", "vimdoc", "python", "bash", "markdown", "markdown_inline" },
```

Comment:
* Lua, query, and vimdoc are used for highlighting on `master` by default (via `vim.treesitter.start()` in a ftplugin)
* Python and bash are pulled in via injections in help files
* Markdown is used for LSP hover on `master`.

Reply:
```suggestion
  ensure_installed = { "c", "lua", "query", "vim", "vimdoc", "python", "bash", "markdown", "markdown_inline" }
```

Is this list supposed to be the same as the list of all "bundled" parser?

nvim 0.9.4 has: c, lua, query, vim, vimdoc
nvim 0.10 has: Plus help, bash, markdown, markdown_line, python

If so, `c` might also need to be added?

Reply:
* `vimdoc` _is_ `help`
* good point about `c`; it's supposed to be the list of all languages for which TS is enabled by default, but then I realized we also need to account for injections (which I forgot `c` for).